model/src/main/proto/spotlight.proto

```diff
@@ -5,55 +5,72 @@ package model;
 option java_package = "org.oppia.android.app.model";
 option java_multiple_files = true;
 
-// Superclass for all the SpotlightViewStates
+// Spotlights are visual elements that are shown during the initial user onboarding experience of
+// the app that highlight certain UI elements and show how/what they should be used for.
 message Spotlight {
   oneof feature {
-    // feature for onboarding screen next button
+    // Corresponds to the onboarding screen's next button.
     SpotlightViewState onboarding_next_button = 1;
 
-    // feature for topic fragment lessons tab
+    // Corresponds to the topic fragment's lessons tab.
     SpotlightViewState topic_lesson_tab = 2;
 
-    // feature for topic fragment revision tab
+    // Corresponds to the topic fragment's revision tab.
     SpotlightViewState topic_revision_tab = 3;
 
-    // feature for first lesson under lessons tab
+    // Corresponds to the first lesson under lessons tab.
     SpotlightViewState first_chapter = 4;
 
-    // feature for promoted stories
+    // Corresponds to the promoted stories.
     SpotlightViewState promoted_stories = 5;
 
-    // feature for back button in exploration screen
+    // Corresponds to the back button in exploration screen.
     SpotlightViewState lessons_back_button = 6;
 
-    // feature for voice over icon in exploration screen
+    // Corresponds to the voice-over icon in exploration screen.
     SpotlightViewState voiceover_play_icon = 7;
 
-    // feature for voice over language options in audio player
+    // Corresponds to the voice-over language option in audio player.
     SpotlightViewState voiceover_language_icon = 8;
   }
 }
 
-// Possible view States for a spotlight
+// Possible view states for a spotlight.
 enum SpotlightViewState {
+  // This is the default state for a spotlight in the beginning. In cases where the app won't be
+  // able to load a spotlight state for a feature, this state shall be reported.
   SPOTLIGHT_VIEW_STATE_UNSPECIFIED = 0;
+
+  // This corresponds to a state when the spotlight for a feature has been seen by the user.
   SPOTLIGHT_SEEN = 1;
+
+  // This corresponds to a state when the spotlight for a feature has been not seen by the user.
```

Comment:
Is this when the spotlight hasn't been seen, or hasn't been dismissed? I think it's worth clarifying here so that readers know how to associate this documentation with what they might observe when using the app.

Reply:
I'm  removing the spotlight state unspecified state as this is currently of no use, and will only confuse developers. Since our current fallback mechanism when the spotlight state can't be retrieved is to show the spotlight again, it is fine to have the default state as "not seen". Per your question, we do not have the "dismissed" state in the app currently.

Reply:
done, please check if the updated doc looks fine

Reply:
Unless I'm mistaken, it doesn't seem like the doc line has been updated. Am I missing something?

Reply:
We do not have the dismiss feature (which was something that was being planned before the full product planning). So this is the state when the feature has not yet been seen by the user.